### ISSUE TITLE
unrecognized option "--sort-view"

### ISSUE THREAD

**Issue by arnisjuraga** [allinurl/goaccess#335]
What I am trying to achieve - is to generate .html report with sorted Bandwith. Like here: #171 

I am trying to execute command:

~~~
goaccess -f full.log  --sort-view=REQUESTS,BY_DATA,DESC  --log-format='%v %h %^[%d:%^] "%r" %s %b "%R" "%u"' --date-format='%d/%b/%Y' --time-format='HH:MM:SS'
~~~

and I get an error:

~~~
unrecognized option '--sort-view=REQUESTS,BY_DATA,DESC'
~~~

When command is executed without sort view - it works ok.

Version:
~~~
GoAccess - 0.9.6.
For more details visit: http://goaccess.io
Copyright (C) 2009-2015 GNU GPL'd, by Gerardo Orellana
~~~

**Comment by aphorise** [comment]
@arnisjuraga - if you refer to the `man goaccess` - I think the option you are after is:
```
--sort-panel= ....
```
no?

**Comment by arnisjuraga** [comment]
The same problem:

~~~
goaccess: unrecognized option '--sort-panel=REQUESTS,BY_DATA,DESC'
~~~

**Comment by allinurl** [comment]
goaccess -f log --sort-panel=REQUESTS,BY_DATA,DESC --log-format='%v %h %^[%d:%^] "%r" %s %b "%R" "%u"' --date-format='%d/%b/%Y' --time-format='HH:MM:SS'

Just tested this as posted above and works. Are you sure you are using two dashes `--`? or any other weird character in between? @aphorise does it work for you?

**Comment by arnisjuraga** [comment]
Hi!

Sorry, tested again, and it's my mistake. I have 2 versions of goaccess on my server. Previously I did 
`--sort-panel`
with 0.8.5,  my bad.

Bu still, please, can You add this option to documentation and correct issue #175 ?

**Comment by aphorise** [comment]
@arnisjuraga regarding your reference to [# 175](https://github.com/allinurl/goaccess/issues/175) - whats the issue? & what exactly are you experiencing? - is it the case that you have special characters that are causing some other crash? - if it is the case - can you please open a separately articulated issue with appropriate details?

@allinurl what about if we keep both arguments: `--sort-view` & `--sort-panel` - it may be good for computability with older builds and matching user expectations with the most recent versions; a related note that I've also been meaning to pitch to you is the reservation of version `0.9.8` specifically as the last version with everything thats old short of new / major features like like `http` views / stats, etc. etc. I'd also advise the indefinite reserve of `0.9.9` for any future commits made to the older base and manual builds made of it which would result in `0.9.9` - this is very similar to how `ssh` was originally and continues. 
What do you think?

**Comment by allinurl** [comment]
@arnisjuraga Thanks for pointing that out. I have updated the docs and issue #171.

@aphorise Probably he meant #171 instead of #175. We can certainly add the alias `--sort-view` to `--sort-panel`. So you are saying release `0.9.8` from `0.9.6` with all the minor stuff that has been pushed lately? I'm working on 1.0.0 which includes major changes.

**Comment by arnisjuraga** [comment]
Thanks! Yes, I meant #171 
Now it works, thanks for the great support!

**Comment by aphorise** [comment]
@allinurl - I'm saying hold off on releasing `0.9.8` as much as possible or do so with the intent of having it as final stable release of / for the past with its  current set of features / as is. Thereafter any serious change or other new feature  or incompatibility with pre `1.0.0` is acceptable and sensible (IMO).

**Comment by aphorise** [comment]
`0.9.9` - as you may have already comprehended - would not be released - as it'd be for all pre `1.0.0` patch / fix that may be applied in the future and which users are required  to build themselves resulting in `0.9.9`.
